JavaScript/JavaScriptSDK/Context/Session.ts

```diff
@@ -122,6 +112,32 @@ module Microsoft.ApplicationInsights.Context {
             }
         }
 
+        private initializeAutomaticSessionWithData(params) {
+            if (params.length > 0) {
```

Comment:
Should line 117 be ideally inside the try catch? We are setting the session id to `params[0]` and then we could possibly be failing to parse the cookie in the `try - catch` below. Is that expected behavior?